src/main/java/edu/utdallas/davisbase/representation/UpdateCommandRepresentation.java

```diff
@@ -0,0 +1,62 @@
+package edu.utdallas.davisbase.representation;
+
+import java.util.List;
+
+public class UpdateCommandRepresentation implements CommandRepresentation {
+
+  private String command;
+  private String table;
+  private List<String> columns;
+  private List<String> values;
+  private WhereExpression whereClause;
+
+  public UpdateCommandRepresentation(String command, String table, List<String> columns, List<String> values, WhereExpression whereClause) {
+    this.command = command;
+    this.table = table;
+    this.columns = columns;
+    this.values = values;
+    this.whereClause = whereClause;
+  }
+
+
+  public String getTable() {
+    return table;
+  }
+
+  public List<String> getColumns() {
+    return columns;
+  }
+
+  public List<String> getValues() {
+    return values;
+  }
+
+  public WhereExpression getWhereClause() {
+    return whereClause;
+  }
+
+  public void setWhereClause(WhereExpression whereClause) {
```

Comment:
Remove the setters to make this object immutable. _(it provides stronger compile-time correctness guarantees, and we don't the setters after construction anyway)_